packages/client/lib/components/ReportForm/FormViews/LocationView.jsx

```diff
@@ -0,0 +1,44 @@
+import useFormStore from '@/store/formStore';
+import { MapPinIcon } from '@heroicons/react/24/outline';
+
+const LocationView = () => {
+  const location = useFormStore((state) => state.location);
+  const setLocation = useFormStore((state) => state.setLocation);
+  const setStep = useFormStore((state) => state.setStep);
+
+
+  return (
+    <div className='flex-1'>
+      <h2 className="text-xl mt-4">Pin map location</h2>
+      <div>
+        <label htmlFor="location" className="block mb-2 text-black">
+          e.g. "Example Road, City" or "-8.54, 115.24"
+        </label>
+        <input
+          type="text"
+          id="location"
+          disabled
+          onChange={(e) => setLocation(e.target.value)}
+          name="location"
+          value={`${location.lat}, ${location.lng}`}
+          placeholder="Provide a location or coordinates"
+          className="block w-full p-2 text-black border border-gray-300 rounded-lg bg-white focus:ring-primary focus:border-primary shadow-md"
+        />
+        <div className="flex justify-center mt-20">
+          <button
+            onClick={(e) => {
+              e.preventDefault();
+              setStep(5);
+            }}
+            className="flex items-center bg-primary hover:primary-700 text-white font-bold py-2.5 px-3.5 rounded-xl"
```

Comment:
```suggestion
            className="flex items-center bg-primary hover:bg-primary/90 text-background font-bold py-2.5 px-3.5 rounded-xl"
```
FYI: `hover:primary-700` is not a utility class but `hover:bg-primary/90` is. We don't have tailwind configured with numbers on our custom color classes. Not sure what you were trying to accomplish with the hover.

Reply:
Imma just remove it. Not sure what I was doing either.